pavelib/assets.py

```diff
@@ -654,6 +660,17 @@ def copy_vendor_library(library, skip_if_missing=False):
         elif not skip_if_missing:
             raise Exception('Missing vendor file {library_path}'.format(library_path=library_path))
 
+    def copy_vendor_library_dir(library_dir, skip_if_missing=False):
+        """
+        Copies all vendor libraries in directory to the shared vendor directory.
+        """
+        library_dir_path = 'node_modules/{library_dir}'.format(library_dir=library_dir)
+        print('Copying vendor library dir: {}'.format(library_dir_path))
```

Comment:
Is this print statement staying in?

Reply:
I guess it might be a little redundant, but I thought it might be helpful to see in the log when it starts to copy everything in a directory vs. just copying one file at a time.